tlatools/org.lamport.tlatools/src/tlc2/tool/liveness/TableauNodePtrTable.java

```diff
@@ -130,19 +131,28 @@ public final long get(final long k, final int tidx) {
 		}
 	}
 
+	public final void put(final long k, final int tidx) {
+		put(k, tidx, UNDONE, UNDONE - 1);
+	}
+	
 	/**
 	 * Add <tidx, elem> into the table. If the table has already contained <k,
 	 * tidx>, overwrite the old value.
 	 */
 	public final void put(long k, int tidx, long elem) {

```

Comment:
Should first sentence of this javadoc be "Add _<k, tidx>_ into the table"?
 1. other comments say that the nodes are identified as `<state, tidx>`, and
 2. the second half of the comment talks about overwriting `<k, tidx>` instead of adding it.

(This question isn't really related to your changes, but I'm only just now getting acquainted with this code.)

Reply:
Clarified the comment.